api/cloud-control/v1beta1/vpcpeering_types.go

```diff
@@ -43,6 +43,15 @@ const (
 	VpcPeeringRemoteNetworkField = ".spec.details.remoteNetwork"
 )
 
+type AwsRouteTableUpdateStrategy string
+
+const (
+	AwsRouteTableUpdateStrategyAuto      = "AUTO"
```

Comment:
That way it's string, it should be as in cloud-resources of the type `AwsRouteTableUpdateStrategy`
```suggestion
	AwsRouteTableUpdateStrategyAuto     AwsRouteTableUpdateStrategy = "AUTO"
```